app/assets/stylesheets/modules/_header.scss

```diff
@@ -56,25 +56,27 @@
   right: 0;
   text-align: center;
   width: 100%;
+}
 
-  input {
-    border-radius: 3px 0 0 3px;
-    width: 438px;
-  }
+.search-archive-field {
+  border-radius: 3px 0 0 3px;
+  margin-right: -4px;
+  width: 438px;
+}
 
-  input[type=submit] {
-    width: 100px;
-    border: none;
-    border-radius: 0 3px 3px 0;
-    font-size: 14px;
-    height: 38px;
-    line-height: normal;
-    padding: 10px;
+.search-archive-submit {
+  width: 100px;
```

Comment:
Properties should be ordered border, border-radius, font-size, height, line-height, margin-left, padding, width